src/server/server.js

```diff
@@ -404,59 +408,11 @@ function processPluginHtml(htmlFile, pluginId, lang) {
     });
 }
 
-function localizeHtmlFile(htmlFile, html, lang) {
-    // Replace localized strings
-    var targetJsonFile = getI18nJsonFile(htmlFile, lang);
-    if (utils.existsSync(targetJsonFile)) {
-        var locItems;
-        try {
-            locItems = require(targetJsonFile);
-        } catch (e) {
-            // Don't care if it fails
-            return;
-        }
-
-        locItems.forEach(function (locItem) {
-            html = replaceLocalizationItem(html, locItem);
-        });
-    }
-    return html;
-}
-
-function replaceLocalizationItem(html, locItem) {
-    // To try and be as targeted as possible, we replace attribute by looking for 'attribute="value"', and
-    // regular text by looking for '>text<'. If we don't find it that way, we revert to a simpler approach.
-
-    var attribute = locItem.attribute;
-    var source, target;
-    if (attribute) {
-        source = attribute + '="' + locItem.source + '"';
-        target = attribute + '="' + locItem.target + '"';
-    } else {
-        source = '>' + locItem.source + '<';
-        target = '>' + locItem.target + '<';
-    }
-    var newHtml = html.replace(source, target);
-    if (newHtml !== html) {
-        return newHtml;
-    }
-
-    // If didn't find it, try something simpler
-    if (attribute) {
-        source = '"' + locItem.source + '"';
-        target = '"' + locItem.target + '"';
-    } else {
-        source = locItem.source;
-        target = locItem.target;
-    }
-    return html.replace(source, target);
-}
-
-function getI18nJsonFile(htmlFile, lang) {
-    var targetJsonFile = htmlFile.split(path.sep);
-    var pos = targetJsonFile.indexOf('src');
-    targetJsonFile.splice(pos + 1, 0, 'i18n', lang);
-    return targetJsonFile.join(path.sep) + '.i18n.json';
+function getLocalizedHtmlFile(htmlFile, lang) {
+    var localizedHtmlFile = htmlFile.split(path.sep);
+    var pos = localizedHtmlFile.indexOf('src');
```

Comment:
Yeah, my thinking was I preferred the localized files all together in a mirrored folder structure, rather than scattered through-out. I've added logic to look under the folder that contains the file if we don't find it in the other location.

This also made me think of something - I now add a "Do not edit this file. It is machine generated" comment to the start of each generated HTML file.